TESTING.md

```diff
@@ -6,41 +6,68 @@ This document describes pointers and notes for testing the GPII infrastructure i
 
 This section contains a tutorial to test the frontend GPII application against the backend. Relevant calls against the backend are noted after each action.
 
-### Testing key-in/key-out and basic settings
+### Testing login/logout and basic settings
 
-1. Obtain a copy of the gpii/windows repository:
+1. Obtain a copy of the gpii/gpii-app repository:
 
 ```bash
-$ git clone git@github.com:GPII/windows.git
+$ git clone https://github.com/GPII/gpii-app.git
+
+```
+
+Note that as of 9/19/2018, the full functionality that works against the cloud only exists on the `feds-audit` branch. Because of this, you should checkout that branch instead of working on master:
+
+```
+$ git checkout feds-audit
 ```
 
-2. Use vagrant to spin up the instance.
+2. Create a new configuration in `configs/app.cloud.json`, containing the following contents:
+
+```
+{
+    "type": "gpii.appWithTaskTray",
+    "options": {
+        "gradeNames": ["fluid.component"],
+        "distributeOptions": {}
+    },
+    "mergeConfigs": [
+        "%gpii-app/configs/app.base.json",
+        "%gpii-universal/gpii/configs/gpii.config.untrusted.development"
+     ]
+}
+```
+
+3. Spin up the vagrant box:
 
 ```bash
-$ cd windows
+$ cd gpii-app
 
 $ vagrant up
 ```
 
-3. Once the windows image is spun up, open a command prompt within the virtual machine and spin up the components locally.
+4. When the machine is up and running, open a command prompt and run the application:
 
-```shell
-$ cd C:\Vagrant
+```
+$ cd c:\vagrant
 
 $ SET GPII_CLOUD_URL=http://flowmanager.<CLUSTER_DNS>
 
-$ SET NODE_ENV=gpii.config.untrusted.development
+$ ./node_modules/.bin/electron . ./configs app.cloud
 
-$ node gpii
+...
 
 ```
 
-The above will bring up a local flowmanager and point it to the specified cloud based flowmanager.
+5. If you are using the `feds-audit` branch, first logout the `noUser` account. This is a hack that exists specifically on that branch:
+
+```
+$ curl http://localhost:8081/user/noUser/logout
+```
 
-4. Once the node process is started, create a new command prompt and login:
+6. Login using a user:
 
-```shell
-$ curl http://localhost:8081/users/carla/login
+```
+$ curl http://localhost:8081/user/carla/login
```

Comment:
It's true that, in the context of this test, there shouldn't be a user keyed in and you won't get into any troubles. However, since you never know if there's a user keyed in on the system, we created the proximityTriggered endpoint for that, see https://github.com/GPII/universal/blob/master/documentation/LoginAndLogoutFlow.md#overview-and-apis. Also, note that at this moment, we need to support the "noUser functionality" in a similar way we have in the feds-audit branch. For that and to keep things simpler, you can simply skip the step 5 (the one that keys out the user noUser) by switching over to the proximityTriggered endpoint when keying carla in. This way, you can key in as carla by issuing ```curl http://localhost:8081/user/carla/proximityTriggered```. This will in fact key out noUser (or any other user) and will key in carla.

Reply:
@javihernandez - Thanks for the explanation! I've removed the extra command and adjusted the instructions to use the `/user/carla/proximityTriggered` resource.